public/api/conf/1.0/testdata/get-pensions-dashboard.md

```diff
@@ -0,0 +1,178 @@
+<table>
+    <thead>
+        <tr>
+            <th>Scenario</th>
+            <th>NINO</th>
+            <th>Response</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th colspan="3">More than 10 qualifying years</th>
+        </tr>
+        <tr>
+            <th colspan="3">Forecast</th>
```

Comment:
As this is test data are we happy to have in lined styling?

Reply:
I'm happy with this using colspan, it's used plenty within other HMRC applications